charts/trento-server/charts/wanda/templates/ingress.yaml

```diff
@@ -0,0 +1,61 @@
+{{- if .Values.ingress.enabled -}}
```

Comment:
I would postpone the ingress implementation maybe, since we want to use the new gateway. Anyway, we could remove it afterwards as well

Reply:
I agree, this should be part of a different story.